### ISSUE TITLE
Suggestion for the README: SQL Connection

### ISSUE THREAD

**Issue by cvrnogueira** [gvenzl/oci-oracle-free#47]
Since SQLcl is running within the Docker host it's easy to see if works. Moreover SQL Developer is running outside so there is not the same challenges. It would be nice to have on the README a session about how to use in SQL Developer

Also, thanks for the project :)

**Comment by gvenzl** [comment]
Hi @cvrnogueira,

Thanks a lot for using these images!

Unfortunately, due to the nature of this being a Docker, Podman, Kubernetes, etc. container, there are many different ways to expose access to the container, and then there are many different tools to connect to an Oracle Database.

Documenting all the different variations would make quite a comprehensive documentation which I am not really qualified to do.

For the most part, all of these should behave as if this is just another Oracle Database running on the local environment, with the exception that port 1521 has to be exposed and not be blocked by a firewall or similar.

For SQL Developer-specific questions, I know that https://twitter.com/thatjeffsmith has always had an open ear for questions and and is eager to help.
